src/nonce-manager/utils.ts

```diff
@@ -0,0 +1,15 @@
+import Contract from 'web3-eth-contract'
+import Web3 from 'web3'
+
+export function encodeNonce(
+    contract: Contract.Contract,
+    address: string
+): string {
+    return contract.methods._nonces(address).encodeABI()
+}
+
+export function decodeNonce(nonceHex: string): string {
+    const web3 = new Web3('')
```

Comment:
Bad idea to create Web3 instance each time. Use `import abiCoder from 'web3-eth-abi'
`. abiCoder.decodeParameter(...)